src/pages/console/data/content-management.mdx

```diff
@@ -46,32 +45,60 @@ When you create a new instance in a table, you can also link it to an instance i
-6. View the details for the *All About Dogs* instance in the *Book* table, and *Martha Rivera* is listed as an author. 
+5. Let's link this book to the *Martha Rivera* author instance that you edited in the previous procedure. Choose **Link to an existing Author**, then choose the ID for *Martha Rivera* from the list of *Author* instances. Note that you have the option to link to an an author because a relationship was defined between the *Book* and *Author* tables during the data modeling process.
+6. View the details for the *All About Dogs* instance in the *Book* table, and *Martha Rivera* is linked as an author. 
+
+Alternatively, you can save save *All About Dogs* before linking to an Author, and instead create the link from an Author record by opening the *Martha Rivera* record, and selecting *All About Dogs* from the related Books.
 
 ## To delete data
 1. Open Studio for an app.
 2. On the **Manage** menu, choose **Content**.
 3. On the **Content** page, select the table to update from the **Select table** menu. For this example, select the *Author* table that you used in the previous procedure.
-4. From the list of records in the table, select the record to delete. For this example, select *Martha Rivera*. On the **Actions** menu, then choose **Delete**.
-5. In the **Delete item** confirmation window, choose **Delete**.
+4. From the list of records in the table, select the checkbox to the left of the record(s) you want to delete. For this example, select *Martha Rivera*. 
+5. On the **Actions** menu, then choose **Delete**.
+6. In the **Delete item** confirmation window, choose **Delete**.
 
 ## To download data
 1. Open Studio for an app.
 2. On the **Manage** menu, choose **Content**.
 3. On the **Actions** menu you have two options for downloading data.
-  1. Choose **Download selected items to CSV** to download only the selected rows of data.
-  2. Choose **Download results to CSV** to download all rows of data on the currently selected page.
+  1. Choose **Download selected items (.csv)** to download only the selected rows of data.
+  2. Choose **Download all items (.csv)** to download all rows of data on the currently selected model.
 4. Once you have selected a download option, your data should immediately start downloading as a CSV.
 
-## Create or edit data in markdown
-You can also edit and store the data as markdown with the markdown editor. This is especially useful for blogs, news, marketing, or content-focused apps where you want the app UI to be styled appropriately. The markdown editor is available by choosing **Edit in markdown** when you are in the **Edit** window for a data instance.
+## Limitations
```

Comment:
Do we also want to add Seed data generation isn’t available if fields are required and of the following type: AWSPhone, Enum, Custom type, and Relationship.

Reply:
additionally, I was thinking of adding the data store limitation. When we have multiple/duplicated tabs with studio the data manager will not load due to indexedDB. cc @abhi7cr for confirmation

Reply:
Added both of these callouts